packages/terra-functional-testing/src/terra-cli/express-server/expressServerHandler.js

```diff
@@ -1,4 +1,4 @@
-const ExpressServer = require('./ExpressServer');
+const ExpressServer = require('../../express-server');
```

Comment:
This will now use the [express-server](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/express-server/express-server.js) that is currently in terra-functional-testing.